### ISSUE TITLE
Hyporeflexia of biceps

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5913]
**Preferred term label:** Hyporeflexia of biceps

**Synonyms**

**Definition (free text, please give PubMed ID)** Reduced intensity of the biceps tendon reflex. 

**Parent term (use hpo.jax.org/app)** Hyporeflexia of upper limbs HP:0012391; Hyporeflexia HP:0001265

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033201
